### PR TITLE
Implement Admin API Key via Configuration

### LINKED ISSUE #67: CentOps: Implement config-based Admin Api Key
As a CentOps Admin
I want to have a pre-configured Admin API Key
So that I can authenticate and call the Admin endpoints on the CentOps API

### Background
Currently, API Key authentication is implemented but it only works for participants using the Public API. This implementation validates API Keys provided in the HTTP header against the participants in the Cosmos collection.

### Acceptance Criteria
- [ ] Should have a separate authentication scheme that also uses ApiKey scheme purely for admins
- [ ] Should use Admin API Key provided through app configuration (environment variable or appsetting)
- [ ] Should return 401 Unauthorized if the API Key does not match
- [ ] Should return 401 Unauthorized if the API Key is missing

### PR DESCRIPTION
**Pull request by decodingahmed**

### CODE REVIEW

File: src/CentOps.Api/Controllers/AdminParticipantController.cs [hunk 1]
```diff
@@ -10,7 +11,7 @@ namespace CentOps.Api.Controllers
 {
     [Route("admin/participants")]
     [ApiController]
-    [Authorize(Policy = "AdminPolicy")]
+    [Authorize(Policy = AuthConfig.AdminPolicy)]
```
Comment (guypritchard) [inline src/CentOps.Api/Controllers/AdminParticipantController.cs:14]: This looks really nice.

**Review by guypritchard — APPROVED** [review]
This is a nice PR - the concept of a different policy for Admin is a really useful one, I think.

**Review by adelechurchill — APPROVED** [review]
Looking good!

File: src/CentOps.Api/Controllers/AdminParticipantController.cs (hunk 1 above)
Comment (decodingahmed) [reply src/CentOps.Api/Controllers/AdminParticipantController.cs:14]: Thank you buddy <3

**Pull request merged**